code/obj/machinery/porters.dm

```diff
@@ -851,6 +851,8 @@ var/global/list/portable_machinery = list() // stop looping through world for th
 		//Products
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/patch/bruise, 20)
 		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/patch/burn, 20)
+		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/mender_refill_cartridge/brute, 5)
+		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/mender_refill_cartridge/burn, 5)
```

Comment:
imo, the porta-nanomed probably shouldn't have mender refills - emergency triage vendor VS doctor restock vendor, yeah?

Reply:
Fair enough. I'll do this, I don't have much opinion on it.